pkg/controller/migmigration/quiesce.go

```diff
@@ -613,28 +648,61 @@ func (t *Task) quiesceCronJobs(client k8sclient.Client) error {
 // Undo quiescence on all CronJobs
 func (t *Task) unQuiesceCronJobs(client k8sclient.Client, namespaces []string) error {
 	for _, ns := range namespaces {
-		list := batchv1beta.CronJobList{}
-		options := k8sclient.InNamespace(ns)
-		err := client.List(context.TODO(), &list, options)
+		cClient, err := t.getSourceClient()
```

Comment:
As above, instead of grabbing src client again here, can we just change the arg type for `unQuiesceCronJobs` from `k8sclient.Client` to `compat.Client`? This is especially relevant here since it won't always be src client. This func is called from `unQuiesceDestApplications` and from `unQuiesceSrcApplications`